### PR TITLE
Restructured sections to flow like the procedure

### PR DESCRIPTION
**Pull request by jawache**
- Moved sections around so they were laid out in the doc like the "steps" section in our into.
- Normalized language to use "quantify", "measure" and "calculate".
- Updated lab-based vs. real-world measurements to use the language of measurements for real-world telemetry and calculation for numbers based on models.

A number of other changes are in here also... appols.

### CODE REVIEW

**Review by atg-abhishek — APPROVED** [review]
LGTM!

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 1]
```diff
@@ -151,37 +140,77 @@ You MUST include an estimate of all the embodied emissions for the hardware used
...
+- scanning
+- build and deploy pipelines
+- testing
+- training ML models
+- operations
+- backup
+- resources to support redundancy
+- resources to support failover
+
+## Functional Unit
```
Comment (SaraEmilyBergman) [inline Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:167]: Add ('R') after "Functional Unit"?

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 2]
```diff
@@ -100,21 +79,31 @@ Where:
...
+
 - Datacenter
 - Individual machine (e.g. VM/Node)
 - Individual service (e.g. API call, ML training job)
-- Execution of code 
+- Execution of code
+
+#### Location-Based Marginal Carbon Intensity (`I`)
+
+The carbon intensity of electricity is a measure of how much carbon (CO2eq) emissions are produced per kilowatt-hour (kWh) of electricity consumed.
```
[the remark this answers is not in the document]
Comment (jawache) [reply Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:97]: I wanted to discuss that today, in another location we use SHOULD but then in the text we talk a lot about how it's the only choice so it feels like a MUST. I didn't feel confident enough to add it to this PR though.

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 3]
```diff
@@ -100,21 +79,31 @@ Where:
 ### Operational Emissions  (`O`) 
 To calculate the operational emissions associate with software, multiply the electricity consumption of the hardware the software is running on by the regional, granular marginal emissions rate. Because this standard uses a consequential approach, marginal emissions rates SHOULD be used for electricity consumption. The marginal emissions rate reflects the change in emissions associcated with a change in demand. 
 
-#### Energy Measurement (`E`) 
-This is a measurement of the energy consumed by a given piece of software for a given task. This MUST be a measurement of energy consumption in kilowatt hours (kWh) of all supporting infrastructure and systems. This could be applied to several taxonomies:  
+#### Energy (`E`)
```
[the remark this answers is not in the document]
Comment (jawache) [reply Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:82]: Is this how I broke it into lines vs. in one paragraph? Or the spacing after the headers (that was my linter complaining in VS code).

[the remark this answers is not in the document]
Comment (SaraEmilyBergman) [reply Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:82]: Yes, the break down. But it is a really minor thing

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md (hunk 1 above)
Comment (jawache) [reply Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md]: Done

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Henry-WattTime) [reply Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:95]: I strongly agree on the must

**Comment by atg-abhishek** [comment]
Approved by WG during the call.

**Pull request merged**